switchbot/devices/lock.py

```diff
@@ -46,6 +60,7 @@ class SwitchbotLock(SwitchbotDevice):
     def __init__(
         self,
         device: BLEDevice,
+        model: LockModel,
```

Comment:
This is a breaking change. It could be a kwarg instead

Reply:
Moved model to kwargs, default to the original lock.